cvmfs/swissknife_sync.cc

```diff
@@ -43,12 +43,14 @@
 #include "platform.h"
 #include "reflog.h"
 #include "sanitizer.h"
+#include "statistics.h"
 #include "sync_mediator.h"
 #include "sync_union.h"
 #include "sync_union_aufs.h"
 #include "sync_union_overlayfs.h"
 #include "util/string.h"
 
+
```

Comment:
Adding and removing empty lines without any reason increases the noise level in the PR. This should be avoided.